packages/node/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@backtrace/node",
+    "version": "0.0.1",
+    "description": "Backtrace-JavaScript Node.JS integration",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "scripts": {
+        "test": "NODE_ENV=test jest"
```

Comment:
we are missing `build`, `rebuild` etc. scripts here

Reply:
done